### PR TITLE
Fixes #96

### LINKED ISSUE #96: No src files present
The tutorials on the sr-ronex github wiki refer to the src files in the sr_ronex_examples/src directory but this does not exist when installing using apt-get under hydro.

### PR DESCRIPTION
**Pull request by YiShadow**
Last tutorial: Accessing the RoNeX data directly from a controller. 
Tested with hardware.

### CODE REVIEW

**Comment by YiShadow** [comment]
Need to add some "install" lines to sr_ronex_examples::CMakeLists.txt to install the items?

File: sr_ronex_examples/src/sr_ronex_simple_controller.cpp [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+void SrRoNeXSimpleController::starting()
+{
+  // Do nothing.
+}
+
+void SrRoNeXSimpleController::update()
+{
+  double position = general_io_->state_.analogue_[0];
```
Comment (toliver) [inline sr_ronex_examples/src/sr_ronex_simple_controller.cpp:62]: What is the goal of this example?
Now it's only reading the analogue 0, but it's not doing anything with this information (other than printing).
(it's OK with me, but maybe we could set some output value?)

**Comment by toliver** [comment]
Yes please, you could add what is suggested in issue #96

**Comment by toliver** [comment]
And update changelog, please. (Forthcoming, etc.)

**Pull request merged**